virt/release_notes/virt-4-20-release-notes.adoc

```diff
@@ -81,10 +80,12 @@ Ensure that you update to the latest z-stream release of your current minor (x.y
 // CNV-62399
 * The descheduler profile `DevKubeVirtRelieveAndMigrate` has been renamed to `KubeVirtRelieveAndMigrate` and is now generally available. The updated profile improves VM eviction stability during live migrations by enabling background evictions and reducing oscillatory behavior. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-enabling-descheduler-evictions.adoc#virt-configuring-descheduler-evictions_virt-enabling-descheduler-evictions[Configuring descheduler evictions for virtual machines].
 
+// CNV-30390
+* vNUMA topology for VMs is now generally available (GA) and is tested and supported for production use. By enabling this feature, you opt in to an improved NUMA configuration for VMs, with better performance and optimal resource allocation. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-NUMA-topology.adoc#virt-NUMA-topology[Working with NUMA topology for virtual machines].
```

Comment:
@abrennan89 do we always write "and is tested and supported for production use." when we write we are GA'ing a product.? i wasn't sure so thought of asking.

Reply:
@abrennan89 probably asking your peers . I don't know.